ComputerInterface/BaseGameInterface.cs

```diff
@@ -87,6 +87,37 @@ public static ETurnMode GetTurnMode()
             return (ETurnMode) Enum.Parse(typeof(ETurnMode), turnMode);
         }
 
+        public static void SetInstrumentVolume(int value)
+        {
+            PlayerPrefs.SetFloat("instrumentVolume", (float)value / 50f);
+            PlayerPrefs.Save();
+        }
+
+        public static float GetInstrumentVolume()
+        {
+            float instVolume = PlayerPrefs.GetFloat("instrumentVolume", 0.1f);
+            return (instVolume);
+        }
+
+        public static void SetItemMode(EItemMode itemMode)
+        {
+            if (GorillaComputer.instance == null) return;
+
+            var itemModeString = itemMode.ToString();
+            PlayerPrefs.SetString("disableParticles", itemModeString);
+            PlayerPrefs.Save();
+            bool disableParticles;
+            if (itemModeString == "TRUE") {disableParticles = true;} else { disableParticles = false;}
```

Comment:
This is equivalent to 
```suggestion
            disableParticles = itemModeString == "TRUE"
```